unit/models/__init__.py

```diff
@@ -36,10 +36,24 @@ def to_dict(self, nested: Optional[bool] = True):
 
 class RelationshipArray(Generic[T], UnitDTO):
     def __init__(self, l: List[T]):
-        self.relationships = l
+        self.data = self.__to_relationships_array(l)
 
-    def to_dict(self):
-        return {"data": list(map(lambda r: r.to_dict(False), self.relationships))}
+    def to_dict(self) -> Dict:
+        return {"data": list(map(lambda r: r.to_dict(False), self.data))}
+
+    def __to_relationships_array(self, l):
```

Comment:
why is it need to be in a separate method and as part of the __init__ method? and if so why double leading underscores and not one?